MembraneRTC/Sources/MembraneRTC/IPC/IPC.swift

```diff
@@ -66,16 +67,21 @@ public class IPCServer: IPC {
             return false
         }
 
-        port = CFMessagePortCreateLocal(nil, name as CFString, { (port: CFMessagePort?, id: Int32, data: CFData?, _: UnsafeMutableRawPointer?) -> Unmanaged<CFData>? in
-            guard let selfObj = port?.associatedSelf() as? IPCServer,
-                  let data = data as Data?
-            else {
+        port = CFMessagePortCreateLocal(
+            nil, name as CFString,
+            {
+                (port: CFMessagePort?, id: Int32, data: CFData?, _: UnsafeMutableRawPointer?) -> Unmanaged<
+                    CFData
+                >? in
```

Comment:
Sometimes there might be a little bit unreadable formatting so you might consider looking briefly at changes and fix where needed. However, becasue there are a lot of files modified we can do this incrementally when implementing next features?